### PR TITLE
refactor: quickstarts flags

### PR DESCRIPTION
**Pull request by jfatta**
Standardize flags definition and usage on the `quickstarts` command.

context: https://auth0team.atlassian.net/browse/CLI-77

### CODE REVIEW

File: internal/cli/quickstarts.go [hunk 1]
```diff
@@ -31,15 +40,21 @@ var (
...
-)
 
-// QuickStart app types and defaults
-const (
-	qsNative    = "native"
-	qsSpa       = "spa"
-	qsWebApp    = "webapp"
-	qsBackend   = "backend"
-	_defaultURL = "http://localhost:3000"
+	clientID = Flag{
```
Comment (Widcket) [inline internal/cli/quickstarts.go:44]: Can you please prefix it with `qs` (`qsClientID`), until we move it to its own package?

File: internal/cli/quickstarts.go [hunk 2]
```diff
@@ -31,15 +40,21 @@ var (
...
+		Name:       "ClientID",
+		LongForm:   "client ID",
+		ShortForm:  "c",
+		Help:       "Client Id of an Auth0 application.",
+		IsRequired: true,
+	}
+
+	stack = Flag{
```
Comment (Widcket) [inline internal/cli/quickstarts.go:52]: Idem above.

File: internal/cli/quickstarts.go [hunk 3]
```diff
@@ -31,15 +40,21 @@ var (
...
-// QuickStart app types and defaults
-const (
-	qsNative    = "native"
-	qsSpa       = "spa"
-	qsWebApp    = "webapp"
-	qsBackend   = "backend"
-	_defaultURL = "http://localhost:3000"
+	clientID = Flag{
+		Name:       "ClientID",
+		LongForm:   "client ID",
```
Comment (Widcket) [inline internal/cli/quickstarts.go:46]:
```suggestion
		LongForm:   "client-id",
```

File: internal/cli/quickstarts.go [hunk 4]
```diff
@@ -31,15 +40,21 @@ var (
...
 
-// QuickStart app types and defaults
-const (
-	qsNative    = "native"
-	qsSpa       = "spa"
-	qsWebApp    = "webapp"
-	qsBackend   = "backend"
-	_defaultURL = "http://localhost:3000"
+	clientID = Flag{
+		Name:       "ClientID",
```
Comment (Widcket) [inline internal/cli/quickstarts.go:45]:
```suggestion
		Name:       "Client ID",
```
The name is used to build the prompt (eg: `Client ID:`).

File: internal/cli/quickstarts.go [hunk 5]
```diff
@@ -31,15 +40,21 @@ var (
...
+		ShortForm:  "c",
+		Help:       "Client Id of an Auth0 application.",
+		IsRequired: true,
+	}
+
+	stack = Flag{
+		Name:      "Stack",
+		LongForm:  "stack",
+		ShortForm: "s",
+		Help:      "Tech/Language of the quickstart sample to download",
```
Comment (Widcket) [inline internal/cli/quickstarts.go:56]:
```suggestion
		Help:      "Tech/Language of the quickstart sample to download.",
```

File: internal/cli/quickstarts.go (hunk 4 above)
Comment (jfatta) [reply internal/cli/quickstarts.go:45]: got it, thanks

File: internal/cli/quickstarts.go [hunk 6]
```diff
@@ -31,15 +40,21 @@ var (
...
+		ShortForm:  "c",
+		Help:       "Client Id of an Auth0 application.",
+		IsRequired: true,
+	}
+
+	stack = Flag{
+		Name:      "Stack",
+		LongForm:  "stack",
+		ShortForm: "s",
+		Help:      "Tech/Language of the quickstart sample to download.",
```
Comment (Widcket) [inline internal/cli/quickstarts.go:56]: Shouldn't it be required as well? (prompt not withstanding)

Comment (jfatta) [reply internal/cli/quickstarts.go:56]: yes, good catch. I was honoring the previous setup but it doesn't make sense without the stack.

**Review by Widcket — APPROVED** [review]
Thanks!

**Pull request merged**